analyzer/compliance-analyzer/src/main/java/eu/fasten/analyzer/complianceanalyzer/Main.java

```diff
@@ -0,0 +1,62 @@
+package eu.fasten.analyzer.complianceanalyzer;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.sql.SQLException;
+
+import org.json.JSONObject;
+import org.json.JSONTokener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import eu.fasten.server.connectors.PostgresConnector;
+import picocli.CommandLine;
+
+
+@CommandLine.Command(name = "ComplianceAnalyzer")
+public class Main implements Runnable{
+
+    private static Logger logger = LoggerFactory.getLogger(Main.class);
+
+    @CommandLine.Option(names = {"-f", "--file"},
+        paramLabel = "JSON",
+        description = "Path to JSON file which contains the repo info")
+    String jsonFile;
+
+    @CommandLine.Option(names = {"-d", "--database"},
+        paramLabel = "dbURL",
+        description = "Database URL for connection",
+        defaultValue = "jdbc:postgresql:postgres")
+    String dbUrl;
+
+    @CommandLine.Option(names = {"-u", "--user"},
+        paramLabel = "dbUser",
+        description = "Database user name",
+        defaultValue = "postgres")
+    String dbUser;
```

Comment:
Those arguments are unused. Can they be deleted?

Reply:
Yes. Done in https://github.com/fasten-project/fasten/commit/7975246450da52793713310dc6123c14c876daf5.